typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -366,14 +366,25 @@ export abstract class HyperlaneAppGovernor<
       };
     }
 
-    // Get the account's config
-    const accountConfig = await this.interchainAccount.getAccountConfig(
-      chain,
-      account.address,
-    );
-    const origin = this.interchainAccount.multiProvider.getChainName(
-      accountConfig.origin,
-    );
+    if (account.address !== icas[chain as keyof typeof icas]) {
+      console.info(
+        chalk.gray(
+          `Account ${account.address} is not a known ICA. Defaulting to manual submission.`,
+        ),
+      );
+      return {
+        type: SubmissionType.MANUAL,
+        chain,
+        call,
+      };
+    }
+
+    const origin = 'ethereum';
+    const accountConfig = {
+      origin,
+      owner: safes[origin],
+    };
```

Comment:
i suppose this is fine for now since we're not governing from this branch, but we'll want to use the `getGovernanceSafes` nomenclature when it's in main
```
const safeOwner = getGovernanceSafes(governanceType)[chain];
```

Reply:
really config shouldnt be hardcoded in here but I will lob that to you 😅